articles/azure/azs-how-create-vm-cli.md

```diff
@@ -0,0 +1,96 @@
+UbuntuServer             Canonical               18.04-LTS          Canonical:UbuntuServer:18.04-LTS:18.04.201808140                         18.04.201808140
+SQL2016SP1-WS2016        MicrosoftSQLServer      Standard           MicrosoftSQLServer:SQL2016SP1-WS2016:Standard:13.1.900310                13.1.900310
+CentOS                   OpenLogic               6.9                OpenLogic:CentOS:6.9:6.9.20180118                                        6.9.20180118
+CentOS                   OpenLogic               7.3                OpenLogic:CentOS:7.3:7.3.20170925                                        7.3.20170925
+CentOS                   OpenLogic               7.5                OpenLogic:CentOS:7.5:7.5.20180815                                        7.5.20180815
+SQL2016SP2-WS2016        MicrosoftSQLServer      Express            MicrosoftSQLServer:SQL2016SP2-WS2016:Express:13.1.900310                 13.1.900310
+WindowsServer            MicrosoftWindowsServer  2016-Datacenter    MicrosoftWindowsServer:WindowsServer:2016-Datacenter:2016.127.20180815   2016.127.20180815
+SQL2016SP2-WS2016        MicrosoftSQLServer      SQLDEV             MicrosoftSQLServer:SQL2016SP2-WS2016:SQLDEV:13.1.900310                  13.1.900310
+SQL2016SP2-WS2016        MicrosoftSQLServer      Standard           MicrosoftSQLServer:SQL2016SP2-WS2016:Standard:13.1.900310                13.1.900310
+SQL2017-SLES12SP2        MicrosoftSQLServer      Enterprise         MicrosoftSQLServer:SQL2017-SLES12SP2:Enterprise:14.0.1000320             14.0.1000320
+SQL2017-SLES12SP2        MicrosoftSQLServer      Express            MicrosoftSQLServer:SQL2017-SLES12SP2:Express:14.0.1000320                14.0.1000320
+SQL2017-SLES12SP2        MicrosoftSQLServer      SQLDEV             MicrosoftSQLServer:SQL2017-SLES12SP2:SQLDEV:14.0.1000320                 14.0.1000320
+SQL2017-SLES12SP2        MicrosoftSQLServer      Standard           MicrosoftSQLServer:SQL2017-SLES12SP2:Standard:14.0.1000320               14.0.1000320
+SQL2017-WS2016           MicrosoftSQLServer      Enterprise         MicrosoftSQLServer:SQL2017-WS2016:Enterprise:14.0.1000320                14.0.1000320
+SQL2017-WS2016           MicrosoftSQLServer      Express            MicrosoftSQLServer:SQL2017-WS2016:Express:14.0.1000320                   14.0.1000320
+SQL2017-WS2016           MicrosoftSQLServer      SQLDEV             MicrosoftSQLServer:SQL2017-WS2016:SQLDEV:14.0.1000204                    14.0.1000204
+SQL2017-WS2016           MicrosoftSQLServer      Standard           MicrosoftSQLServer:SQL2017-WS2016:Standard:14.0.1000320                  14.0.1000320
+```
+
+After you have decided on which image you wish to use, run the following command, substituting your own variables where applicable. The **az vm create** command will also create any resources required for the VM (for example, a network interface card, a network security group, et cetera)

```

Comment:
After you've decided which image you want to use, run the following command, substituting your own variables where applicable.

Use backticks (`) around az vm create rather than bold markdown

Bonus points for spelling out et cetera in full, but better to use "and so on" instead